### DISCUSSION TITLE
[VIDEO] Persisting Livewire State to LocalStorage using Alpine.js

### DISCUSSION THREAD

**Discussion by jonneroelofs** [livewire/livewire#4661] [2. Show and tell]
I have created a video showing the power of combining Livewire's `@entangle` with Alpine's $persist to very easily sync state between Livewire and LocalStorage.

[Persisting Livewire State to LocalStorage using Alpine.js](https://youtu.be/KwE3mf3seoU)

**Comment by joshhanley** [comment]
Nice video!

Maybe in your comment above, put the entangle tag in back ticks, so you're not pinging the person who has the entangle handle! 😬

**Reply by jonneroelofs** [reply]
haha good advice

**Comment by xxdalexx** [comment]
Does this result in a double request for the initial page load? Meaning the initial with the backend defaults, and then a second one once the localstorage variables are synced?
